code/modules/projectiles/guns/projectile/constructable/railgun.dm

```diff
@@ -291,7 +291,25 @@
 	loadedammo.throw_at(target,distance,speed)
 	loadedammo = null
 
+/obj/item/weapon/gun/projectile/railgun/preloaded
+	var/ammotype = /obj/item/weapon/coin/iron
+	var/capacitortype = /obj/item/weapon/stock_parts/capacitor/adv/super
+	
+/obj/item/weapon/gun/projectile/railgun/preloaded/New()
+	..()
+	loadedassembly = new /obj/item/weapon/rail_assembly
+	rails_secure = 1
+	loadedammo = new ammotype
```

Comment:
Why isn't this `new ammotype(src)` ? Right now, it looks like this ammo is stored in nullspace.

Reply:
lemme check that